doc/tutorials/introduction/linux_install/linux_install.markdown

```diff
@@ -65,6 +65,9 @@ Building OpenCV from Source Using CMake
     For example
     @code{.bash}
     cmake -D CMAKE_BUILD_TYPE=Release -D CMAKE_INSTALL_PREFIX=/usr/local ..
+@note
```

Comment:
You should not add a `@note` inside `@code ... @endcode` block, it will not work this way. You have to add it after `@endcode`.